app/assets/javascripts/application.js

```diff
@@ -12,10 +12,8 @@
 //= require_tree ./modules
 //= require set-ga-client-id-on-form
 
-jQuery(function ($) {
-  var $form = $('.js-service-sign-in-form')
+var form = document.getElementsByClassName('js-service-sign-in-form')
```

Comment:
The tests are passing so this seems to be working but I'm slightly confused about something. Both the original jQuery and the replacement (`getElementsByClassName`) return an array-like object of elements to act on, but then this is treated as a single entity when the attributes are read and set.

I think this works in the test because the test bypasses this bit of code to pass a single form element directly to the module. Have you tested this works as expected on a locally running page in `government-frontend`? I think there might also be value in having a test that simulates more than one form on a page, to check it applies the right attribute to all forms.

Reply:
Blimey, yeah I think this would have broken in prod. Just a product of the disconnect between how the thing's called in tests and production. I'll see if I can work out a way to test it better